src/app/home/home.component.css

```diff
@@ -0,0 +1,24 @@
+.home-container{
+    display: flex;
+    flex-direction: row;
+}
+
+.main-text-box{
+    align-items: center;
+    
+    width: 610px;
+    height: 180px;
+    margin-left: 50px;
+    margin-top: 100px;
+    font-family: 'Poppins';
```

Comment:
you can't add this width this way. it will not be responsive

Reply:
done!